geometry_msgs/Point[] camera_waypoints